.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - 'draft'
+      - 'main'
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+env:
+  NODE_VERSION: '22'
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js
```

Comment:
do we need install pnpm and can we benefit from pnpm-lock of same hash with pnpm step?

https://github.com/fractaldotbox/geist-dapp-kit/blob/draft/.github/workflows/check.yml#L45

Reply:
Thanks, that's a good catch. I have updated the workflows and they now share a composite action that sets up node and pnpm using recommended caching approach.